examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -303,6 +303,13 @@ class ReportCommand : public ModelCommand, public chip::app::ReadClient::Callbac
         return mReadClient->SendRequest(params);
     }
 
+    // Use a 3x-longer-than-default timeout because wildcard reads can take a
+    // while.
+    chip::System::Clock::Timeout GetWaitDuration() const override
+    {
+        return mTimeout.HasValue() ? chip::System::Clock::Seconds16(mTimeout.Value()) : (ModelCommand::GetWaitDuration() * 3);
```

Comment:
ModelCommand::GetWaitDuration() is hard-coded to 10 seconds. So now the default would be 3*10 seconds instead of originally 10 seconds, right? Is there any rationale behind the new value other than being longer than 10 seconds?

Reply:
>  Is there any rationale behind the new value other than being longer than 10 seconds?

I timed how long it takes to do an "all endpoints, all clusters, all attributes" read for an m5stack running all-clusters app, and it was ~15 seconds.  If figured twice that was a reasonable default that should at least usually work...

I considered just hardcoding 30, but figured we might want it to scale if the ModelCommand timeout gets changed for some reason.

But I'm pretty open to changing exactly what the default value is and how it's computed if there are better suggestions.

Reply:
Well, yes, your values seem to be quite reasonable. I tested it on our Nordic-based devices and there it took about 12 seconds in worst case.
I guess that such transfers might take way longer on real-life Thread networks due to multi-node-hopping